GrandCentralBoard/Widgets/Watch/View/WatchWidgetView.swift

```diff
@@ -47,16 +36,10 @@ final class WatchWidgetView: UIView, ViewModelRendering {
     override func awakeFromNib() {
         super.awakeFromNib()
 
-        meetingName.text = ""
-        meetingETA.text = ""
-        startsIn.text = ""
-        hourLeft.text = ""
-        hourRight.text = ""
-        calendarName.text = ""
-
-        blinkingImageFlashingAnimationController = FlashingAnimationController(view: blinkingImage, interval: pulsatingInterval,
-                                                                               alphaDepth: semiTransparentAlpha)
-        blinkingImageFlashingAnimationController?.startFlashing()
+        watchFaceImage.image = nil
+        centeredTimeLabel.text = nil
+        alignedTimeLabel.text = nil
+        eventLabel.attributedText = nil
```

Comment:
Yes, I want to clear the `view` state to not display Storyboard placeholders before the first `render(viewModel: ViewModel)` call gets performed (and it gets called asynchronously, with clock's widget specific interval). Storyboard placeholders are good for view maintenance.